src/foam/nanos/auth/User.js

```diff
@@ -229,36 +228,47 @@ foam.CLASS({
       section: 'administrative'
     },
     {
-      class: 'FObjectProperty',
-      of: 'foam.nanos.auth.Phone',
-      name: 'phone',
+      class: 'String',
```

Comment:
We have a `foam.core.PhoneNumber` property that we should be taking advantage of.

```suggestion
      class: 'PhoneNumber',
```